tests/unit/test_qt.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from nxdrive.gui.folders_dialog import get_regexp_validator
+
+
+@pytest.mark.parametrize(
+    "input_data",
+    [
+        "simple_test_flder",
+        "SIMPLE SPACED #1",
+        "[WARNING]",
+        "STRANĜE CHARS",
+        "LIFE'S STRANGE",
+    ],
+)
+def test_regexp_validator_should_pass(input_data):
+    validator = get_regexp_validator()
+    assert validator.validate(input_data, 0)[0] == 2  # Acceptable
```

Comment:
You should add `qt/constants.py::Acceptable` (https://doc.qt.io/qt-5/qvalidator.html#State-enum) and using it here.